src/20_Components/amp-lightbox-gallery.html

```diff
@@ -0,0 +1,50 @@
+<!---{
+    "experiments": ["amp-lightbox-gallery"],
+    "preview": "default"
+}--->
+<!--
+  ## Introduction
+
+The amp-lightbox-gallery component provides a "lightbox” experience for AMP components (e.g., amp-img, amp-carousel). When the user interacts with the AMP element, a UI component expands to fill the viewport until it is closed by the user. Currently, only images are supported.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="/components/amp-lightbox-gallery/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <title>amp-lightbox-gallery</title>
+  <!-- ## Setup -->
+  <!-- Import the amp-lightbox-gallery component in the header. Only import amp-carousel if you are using a carousel. -->
```

Comment:
can you add 2 different comments for the 2 imports? otherwise the second one will not appear in the doc.

Reply:
Oh. Gotcha. Thank you!